app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_permitted_parameters
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
+  end
+end
```

Comment:
- [ ] The previous error still persists as shown below even though the name field is specified and I've given a value it seems the `name` parameter is not permitted in the registration controller so kindly make sure to fix this.
- Here's how it behaves:
  ![firefox_zo8RqxKyC4](https://github.com/mohammadnewton/blog_app/assets/104968236/b08cf5e3-4bb6-41ce-9556-289b5d6bce39)
- Here's the error being shown in the server side:
  ![image](https://github.com/mohammadnewton/blog_app/assets/104968236/0876ddf5-8344-495b-b635-8a97aaa430b0)

Reply:
- Well resolved 👍🏽 I was able to properly sign up.
- [ ] [OPTIONAL] However, a part of the requirements states that `:confirmable` should be enabled in the user model but you did not do that. With `:confirmable`, users are required to verify their email addresses after signing up. This adds an extra layer of security and ensures that the provided email addresses are valid.
- Kindly implement this fix!
<img width="1374" alt="Screenshot 2023-11-17 at 13 15 51" src="https://github.com/mohammadnewton/blog_app/assets/58692590/344f6477-e163-4668-b021-06c48988d203">